### ISSUE TITLE
x64 SFM?

### ISSUE THREAD

**Issue by Joose2005** [ValveSoftware/Source-1-Games#5556]
With SFM models becoming higher in memory usage and such I feel like an x64 build would be very helpful for animators and would significantly reduce crashes. Other source games like Garrys Mod and TF2 have (from what I've heard) an x64 branch. Since SFM is built on TF2 I see no reason for SFM to not receive the same or at least a similar upgrade. Not to mention that SFM and TF2 were (guessing by release dates) compiled with the 2007 Source SDK. Even so, Garrys mod was initially released in 2006 which suggests it was made in he 2006 SDK.

There was also at one point a x64 release of Half-Life from the information I could find which proves that the source engine is capable of 64 bit in some form.

**Comment by Joose2005** [comment]
how do I add labels?

**Comment by JoriKos** [comment]
Someone else does the labels. I doubt that SFM really counts as a "source 1 game", so this is probably not the right place to ask.

**Comment by Joose2005** [comment]
> Someone else does the labels. I doubt that SFM really counts as a "source 1 game", so this is probably not the right place to ask.

Even though it's not a game it was still made in Source 1 and.

**Comment by gaelcoral** [comment]
It would be nice to see X64 SFM, but I doubt very much.

**Comment by Joose2005** [comment]
> It would be nice to see X64 SFM, but I doubt very much. At this point I just want to see S2FM in an official form without needing source 2 games

Yeah, but I'm just saying that if we're updating other source 1 applications to X64, we should include SFM.

**Comment by Joose2005** [comment]
TF2 has been updated to x64, I still see no reason for SFM to not get the same treatment. If it doesn't happen, I'll download TF2 and will try to find a way to do it myself.

**Comment by AlexisSCD** [comment]
SFM could really benefit from the move to a x64 based environment, it would boost performance for SFM and also being able to use a far wider range of memory. As someone mentioned S2FM, the current state of Source 2 is still far from completion and most likely won't be public for a few more years. Also, Source 1 assets are mostly proprietary, and Source 2 uses entirely new filetypes. I believe Valve has their own program that helps automatically port Source 1 assets to Source 2, but idk if they'll make it public.

**Comment by H-O-W-L-45** [comment]
With the recent update of other old Source Engine games and the release of the x64 source code, an update for SFM would be very much appreciated.

**Comment by ChaoticUltimateJ** [comment]
After doing a while of digging I found that SFM is built on the Alien Swarm branch, which isn't on the 2013 engine, which probably means that SFM would have to basically be remade entirely for the 2013 engine. This is mostly speculation though, since I don't know anything about the code behind SFM, Alien Swarm, or the Source Engine, so I have no idea how much of it can just be dragged over and work.

**Comment by windows10win7posreadywoeskio** [comment]
Original SFM was built on the 2007/orangebox engine so if i had to take a guess they could still port SFM to 2013 Engine, it's just it would probably take alot of time remaking the improvements or whatever
